src/features/click-to-load.js

```diff
@@ -597,11 +618,38 @@ function replaceYouTubeCTL (trackingElement, widget) {
         // Block YouTube embedded video and display blocking dialog
         widget.autoplay = false
         const oldPlaceholder = widget.placeholderElement
-        const { blockingDialog, shadowRoot } = createYouTubeBlockingDialog(trackingElement, widget)
-        resizeElementToMatch(oldPlaceholder || trackingElement, blockingDialog)
-        replaceTrackingElement(widget, trackingElement, blockingDialog)
-        showExtraUnblockIfShortPlaceholder(shadowRoot, blockingDialog)
-        hideInfoTextIfNarrowPlaceholder(shadowRoot, blockingDialog, 460)
+
+        if (isMobileApp) {
+            const mobileBlockedPlaceholder = new DDGCtlPlaceholderBlocked({
```

Comment:
It's cool how you've set this up, but it took me a few reads to grok. I think a comment to explain usage for the constructor would help as I mentioned elsewhere. Also though, perhaps ending the class name with "Element" would help make it clearer that this class inherits from HTMLElement too?